ironhawk/main.go

```diff
@@ -17,6 +17,7 @@ var (
 	boldGreen = color.New(color.FgGreen, color.Bold).SprintFunc()
 	boldRed   = color.New(color.FgRed, color.Bold).SprintFunc()
 	boldBlue  = color.New(color.FgBlue, color.Bold).SprintFunc()
+	inWatchMode bool
```

Comment:
instead of sating `isWatchMode`, let's have a var called `mode` with possible values `command` and `watch`. This will keep it consistent with our marine database engine as well.